vulcan/lib/client/jsx/components/workflow/user_interactions/outputs/plot.jsx

```diff
@@ -34,8 +34,7 @@ const PlotOutput = ({data: plots}) => {
                 zerolinewidth: 1,
                 linecolor: '#000'
               },
-              width: 800,
-              height: 800,
+              autosize: true,
```

Comment:
Hm, I thought UMAPs had to be square. Does changing this affect the UMAP dimensions and thus utility?

Reply:
Err, forgot about this change, I made this while fiddling with the thumbnail image - I will pull it out. While it's true the UMAP looks best square, I don't think this assumption is true for plots in general. What would be nice is if the workflow author could define a fixed aspect ratio and the user could scale the plot size responsively by scaling the window as they liked. But I don't think this quite works that way yet.

Reply:
Perhaps even a per-workflow **suggested** aspect-ratio, but then we let the user adjust even around that?  Thinking of the general plotting workflow as an example, a y_plot with 3 groups on the x-axis would need a different aspect ratio than one with 20, but it's the same workflow.cwl, so it'd be nice to give that 'complete' flexibility.